posthog/models/prompt.py

```diff
@@ -57,470 +40,554 @@ class Meta:
-                "placement": "top-start",
-                "reference": "annotations-table",
-                "icon": "annotations",
-            },
-            {
-                "step": 1,
-                "type": "tooltip",
-                "text": "Annotations can be really helpful later down the road, especially for new team members. Getting into the habit of adding annotations will help you get the most out of PostHog.",
-                "placement": "top-start",
-                "reference": "create-annotation",
-                "icon": "annotations",
-            },
-        ],
-        "rule": {"path": {"must_match": ["/annotations"]}, "must_be_completed": ["start-flow"]},
-        "type": "product-tour",
-    },
-    {
-        "key": "apps-product-tour",
-        "prompts": [
-            {
-                "step": 0,
-                "type": "tooltip",
-                "text": "PostHog has a growing library of apps to help you import, export or transform data. Once you’ve installed an app, click the blue gear icon to configure it.",
-                "placement": "top-start",
-                "reference": "apps-tabs",
-                "icon": "apps",
-            },
-            {
-                "step": 1,
-                "type": "tooltip",
-                "text": "Apps in PostHog function as a chain, with one acting after another before data is stored. Here, you can drag enabled apps to run in any order you’d like.",
-                "placement": "top-start",
-                "reference": "apps-tabs",
-                "icon": "apps",
-            },
-            {
-                "step": 2,
-                "type": "tooltip",
-                "text": "Can’t find the app you need? Everything on PostHog is open source, so use the Advanced tab to get started building and releasing your own apps to the community!",
-                "placement": "top-start",
-                "reference": "apps-tabs",
-                "icon": "apps",
-            },
-        ],
-        "rule": {"path": {"must_match": ["/project/apps"]}, "must_be_completed": ["start-flow"]},
-        "type": "product-tour",
-    },
-    {
-        "key": "toolbar-product-tour",
-        "prompts": [
-            {
-                "step": 0,
-                "type": "tooltip",
-                "text": "If you’re signed in to PostHog, the toolbar enables you to make some changes right within your product. Just authorize a domain, enable the toolbar and away you go!",
-                "placement": "top-start",
-                "reference": "toolbar-authorized-toggle",
-                "icon": "apps",
-            }
-        ],
-        "rule": {"path": {"must_match": ["/toolbar"]}, "must_be_completed": ["start-flow"]},
-        "type": "product-tour",
+        "rule": {
+            "path": {"must_match": ["/*"], "exclude": ["/ingestion", "/ingestion/*"]},
+            "must_be_completed": ["start-flow"],
+        },
+        "type": "one-off",
     },
+    # {
```

Comment:
I wonder if there's a better way than commenting out the code here...

Either a tag/branch to keep it easily discoverable. Or a comment so the future traveller understands why there's a block of commented code

Reply:
You're right, I'll remove it and add it on a different branch!